proxy/router.go

```diff
@@ -123,6 +124,13 @@ func (router *Router) customModResponse(r *http.Response) error {
 				return err
 			}
 		}
+
+		// requests from the client also need to be processed here,
+		// otherwise this 302 status will be sent to the client unchanged
+		r.StatusCode = http.StatusOK
```

Comment:
If we change the status code to OK, maybe we need to write `resp.Body` to `r.Body`.

Reply:
Well, I think the OK status only means that the service has been successfully downloaded, this place and then put a body belongs to the redundant

Reply:
https://github.com/goproxyio/goproxy/blob/master/proxy/router.go#L118